crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -800,7 +804,10 @@ impl Binary {
             }
             BinaryOp::Div => {
                 if r_is_zero {
-                    todo!("Panic - division by zero");
+                    return Err(RuntimeError {
+                        location: None,
```

Comment:
Definitely looks like a RuntimeError constructor taking only a RuntimeErrorKind could be useful here. Perhaps even a `impl From<RuntimeErrorKind> for RuntimeError` where the location defaults to None. We could slowly remove these over time then once we have more locations threaded through SSA.

Reply:
Done